process/process.lua

```diff
@@ -367,11 +370,20 @@ function process.handle(msg, _)
       function (msg)
         return msg.Tags.Type == "Process" and Owner == msg.From 
       end,
-      require('.boot')(ao)
+      function ()
```

Comment:
Technically we should be accepting a message here and passing it like:
```
require('.boot')(ao)(msg)
```
The boot file uses it to boot from eval data or weave drive, and also to do the initial AOS hello world message via the Inbox. We don't NEEEEED any of that, so technically we're ok, but if we wanted to match the original behaviors of AOS, we'd do as outlined above.

Reply:
a3f3523